includes/widgets-admin/elementor/elementor_widget_manager.php

```diff
@@ -31,6 +31,73 @@ public function init() {
 		// Register Orbit Fox Elementor Widgets
 		add_action( 'elementor/widgets/widgets_registered', array( $this, 'register_elementor_widget' ) );
 
+		// Before Elementor Widget Settings Save
+		add_filter( 'elementor/document/save/data', array( $this, 'before_settings_save' ), 10, 2 );
+
+	}
+
+	/**
+	 * Sanitize the field attributes for whe width attribute.
+	 *
+	 * @param array $field The field attributes.
+	 *
+	 * @return array
+	 */
+	private function sanitize_field_attributes( $field ) {
+		$address_fields = array( 'addr2', 'city', 'state', 'zip', 'country' );
+		foreach ( $address_fields as $attribute ) {
+			if ( isset( $field[$attribute . '_width'] ) ) {
+				$field[$attribute . '_width'] = is_numeric($field[$attribute . '_width'] ) ? $field[$attribute . '_width'] : '100';
+			}
+		}
+		return $field;
+	}
+
+	/**
+	 * Search and modify the widget settings.
+	 *
+	 * @param array $elements_data The elements data.
+	 */
+	private function search_and_modify_widget_settings( &$elements_data ) {
```

Comment:
This can be more descriptive. Like

```php
/**
 * Recursively sanitize form fields from Widgets.
 *
 * @param array $elements_data The elements data.
 */
private function search_and_sanitize_widget_form_fields( &$elements_data ) {
```